pypredict/templates/shared/base.html

```diff
@@ -20,6 +20,7 @@
     <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/js/bootstrap.bundle.min.js"
         integrity="sha384-b5kHyXgcpbZJO/tY9Ul7kGkf1S0CWuKcCD38l8YkeH8z8QjE0GmW1gYU5S9FOnJ0"
         crossorigin="anonymous"></script>
+    <script src="{{ url_for('static', path='js/config.js') }}"></script>
```

Comment:
For some reason, I had to add the script in both places in order for the app to work without docker.

Reply:
This is still true even after ensuring that I have activated the virtual environment prior to running locally.